### PR TITLE
README: Add Communication section with Forum information

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README. Similar PRs are being raised across all included collections under the ansible-collection org for now.

- If you have your forum group and/or tags related to the collection, please update corresponding lines by suggesting changes to the PR.
- If the collection is not present on the Ansible forum yet, please check out the existing [tags](https://forum.ansible.com/tags) and [groups](https://forum.ansible.com/g) - use what suits your collection. If there is no appropritate tag and group yet, please [request one](https://forum.ansible.com/t/requesting-a-forum-group/503/17). Then update corresponding lines by suggesting changes to the PR.
- Presence in the forum will soon likely become a part of the Collection inclusion requirements.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

**Comment by Andersson007** [comment]
I don't think the CI failure is related, so ready for review

File: README.md [hunk 1]
```diff
@@ -7,6 +7,19 @@ https://dev.azure.com/ansible/ansible.posix/_apis/build/status/CI?branchName=mai
 <!-- Describe the collection and why a user would want to use it. What does the collection do? -->
 An Ansible Collection of modules and plugins that target POSIX UNIX/Linux and derivative Operating Systems.
 
+## Communication
+
+* Join the Ansible forum:
+  * [Get Help](https://forum.ansible.com/c/help/6): get help or help others.
+  * [Posts tagged with 'your tag'](https://forum.ansible.com/tag/YOUR_TAG): subscribe to participate in collection-related conversations.
+  * [Refer to your forum group here if exists](https://forum.ansible.com/g/): by joining the team you will automatically get subscribed to the posts tagged with [your group forum tag here](https://forum.ansible.com/tags).
```
Comment (Andersson007) [inline README.md:15]:
```suggestion
```

**Comment by mariolenz** [comment]
@Andersson007 How about a new release to see the new communication section on [galaxy](https://galaxy.ansible.com/ui/repo/published/ansible/posix/docs/)?

cc @Akasurde @maxamillion

**Comment by Andersson007** [comment]
@saito-hideki what do you think ^?

**Comment by saito-hideki** [comment]
@Andersson007 @Akasurde @maxamillion 
We haven't had a release for a while, so September is a good time. If there are no objections, I have time to do it next week; let me know what you all think.

**Comment by Andersson007** [comment]
> @Andersson007 @Akasurde @maxamillion We haven't had a release for a while, so September is a good time. If there are no objections, I have time to do it next week(Sep 12); let me know what you all think.

@saito-hideki SGTM, thanks!
@mariolenz thanks for pinging!

**Comment by maxamillion** [comment]
Sounds good! shipit

**Comment by mariolenz** [comment]
> We haven't had a release for a while, so September is a good time. If there are no objections, I have time to do it next week(Sep 12)

@saito-hideki Could you please close #559 when you've published the new release?

**Comment by saito-hideki** [comment]
@mariolenz Sure will do. Thanks! :)

**Pull request merged**